### PR TITLE
Healthbot - Update Python.md based on az cli extension doc & Create Java.md file.

### PR DESCRIPTION
**Pull request by guy-microsoft**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [x] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet. **Jan 3rd 2021**
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet. **Today if possible**
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [x] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by guy-microsoft** [comment]
1. Created a Java.ms file based on [this comment](https://github.com/Azure/sdk-release-request/issues/1095#issuecomment-752366171).
2. Updated Python.md based on az cli extension documentation.

Ready for review.

**Comment by qiaozha** [comment]
@ChenTanyi Does the readme.java.md look good to you ?

**Comment by qiaozha** [comment]
@guy-microsoft Hi, Considering healthbot is a new service. I think it's better to add the python track2 configuration as well. You can change the readme.python.md  following the sample here https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/documentation/samplefiles/readme.python.md

**Comment by guy-microsoft** [comment]
Thanks @qiaozha, added.

File: specification/healthbot/resource-manager/readme.python.md [hunk 1]
```diff
@@ -3,19 +3,42 @@
...
+```yaml $(python) && $(track2)
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.healthbot
+  package-name: azure-mgmt-healthbot
+  package-version: 2020-12-08
   clear-output-folder: true
 ```
```
Comment (qiaozha) [inline specification/healthbot/resource-manager/readme.python.md:25]:
```suggestion
```yaml $(python) && $(track2)
azure-arm: true
license-header: MICROSOFT_MIT_NO_VERSION
payload-flattening-threshold: 2
namespace: azure.mgmt.healthbot
package-name: azure-mgmt-healthbot
package-version: 2020-12-08
clear-output-folder: true
```
```

File: specification/healthbot/resource-manager/readme.python.md [hunk 2]
```diff
@@ -3,19 +3,42 @@
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
-```yaml $(python)
+```yaml $(python) && !$(track2)
```
Comment (qiaozha) [inline specification/healthbot/resource-manager/readme.python.md:6]:
```suggestion
```yaml $(python) && !$(track2)
python-mode: create
```
I think the sample file missing `python-mode: create`

File: specification/healthbot/resource-manager/readme.python.md (hunk 1 above)
Comment (qiaozha) [reply specification/healthbot/resource-manager/readme.python.md:25]: Also need add a `python-mode: create` here.

File: specification/healthbot/resource-manager/readme.python.md [hunk 3]
```diff
@@ -3,19 +3,42 @@
...
+license-header: MICROSOFT_MIT_NO_VERSION
+payload-flattening-threshold: 2
+namespace: azure.mgmt.healthbot
+package-name: azure-mgmt-healthbot
+package-version: 2020-12-08
+clear-output-folder: true
+```
 
-```yaml $(python)
+``` yaml $(python) && $(python-mode) == 'update'
```
Comment (qiaozha) [inline specification/healthbot/resource-manager/readme.python.md:27]:
```suggestion
``` yaml $(python) && $(python-mode) == 'update' && !$(track2)
```

**Review by ChenTanyi — APPROVED** [review]
Java readme is OK

File: specification/healthbot/resource-manager/readme.python.md [hunk 4]
```diff
@@ -3,19 +3,42 @@
...
+python:
+    no-namespace-folders: true
+    output-folder: $(python-sdks-folder)/{svcName}/azure-mgmt-healthbot/azure/mgmt/healthbot
+```
+``` yaml $(python) && $(python-mode) == 'create'
```
Comment (qiaozha) [inline specification/healthbot/resource-manager/readme.python.md:32]:
```suggestion
``` yaml $(python) && $(python-mode) == 'create' && !$(track2)
```

**Comment by xccc-msft** [comment]
@guy-microsoft You might want to add title like [this](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/compute/resource-manager#basic-information). According to [autorest generic-extensions](https://azure.github.io/autorest/extensions/#generic-extensions), the setting of [x-ms-code-generation-settings](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2020-12-08/healthbot.json#L7) is already deprecated.

**Comment by guy-microsoft** [comment]
According to @00Kai0, only track2 is needed for new services, so I'm reverting the changes for python.md file. Only changes to java.md remain, which were approved.

**Comment by ChenTanyi** [comment]
@guy-microsoft As @xccc-msft said, you might need to add title in `java.md`. Otherwise, the generation will fail.

**Comment by guy-microsoft** [comment]
@ChenTanyi When you say "title in java.md", what do you mean?
I looked at @xccc-msft comment now and it talks about readme.md file..

**Comment by ChenTanyi** [comment]
They are the same, you can see https://github.com/azure/azure-rest-api-specs/blob/a788294beecc16b21b4713d4e3af198476464a68/specification/containerservice/resource-manager/readme.java.md#L11

**Comment by guy-microsoft** [comment]
@ChenTanyi Ok, but I'm still not sure what I should add to the readme.md file? Something like that?
image

**Comment by ChenTanyi** [comment]
@guy-microsoft yes, you need to add a `Java` section in the `readme.md`

**Comment by guy-microsoft** [comment]
@ChenTanyi Thanks for clarifying, I added the Java section. Please let me know it's fine.

**Comment by ChenTanyi** [comment]
@guy-microsoft You still didn't add `title`

**Review by xseeseesee — APPROVED** [review]

**Comment by guy-microsoft** [comment]
Thanks @xccc-msft, this PR can be merged.

**Review by lirenhe — APPROVED** [review]

**Pull request merged**